### DISCUSSION TITLE
Do apps need to call `close` on isar instances

### DISCUSSION THREAD

**Discussion by ronw98** [isar/isar#1377] [Help] [answered]
I've seen that the `Isar` class has a `close` method that releases the instance it is called on, but I haven't seen any example in the documentation nor in the sample app of closing an isar instance.

Assuming my app regularly reads from/writes to the same Isar database, do I need to open, read/write, close each time I want to access it or just opening the instance at app startup and letting the system close it when the app is closed is enough ?

**Comment by simc** [comment, marked as answer]
You don't need to close the instance. This is only required if you get a copy of your instance in a background isolate. Then you have to close this copy when the isolate exits.

**Reply by ronw98** [reply]
alright thanks a lot for your answer !
